modules/api/php/views/visit/dicoms.class.inc

```diff
@@ -52,8 +52,9 @@ class Dicoms
     private static function _formatDicomTars(DicomTarDTO $dicom): array
     {
         return [
-            'Tarname'    => $dicom->getTarname(),
-            'SeriesInfo' => array_map(
+            'Tarname'     => $dicom->getTarname(),
+            'Patientname' => $dicom->getPatientname(),
```

Comment:
This is changing the result for both 0.0.3 and 0.0.4-dev, it needs a version check.